Snake/Snake/UILayer.cs

```diff
@@ -91,12 +95,36 @@ static void endGame()
             System.Console.WriteLine("Enter r to replay.");
 
             UserReplay = Console.ReadLine().ToString();
-            if (UserReplay == "R" || UserReplay == "r")
+            if (UserReplay.Equals("r", StringComparison.CurrentCultureIgnoreCase)) 
             {
                 ExitGame = false;
                 Console.Clear();
                 Main(null);
             }
         }
+
+        static void initialMenuLoad()
+        {
+            do
+            {
+                
+                Style.menuImage();
+                try
+                {
+                    gameMode = Convert.ToInt32(Console.ReadLine());
+                }
+                catch
+                {
+                    Console.WriteLine();

```

Comment:
what would you do if you caught it? seems like you are screwed anyway. Consider how a higher up global exception handler could help instead and don't try catch these lower down exceptions unless you can really do something about it at that lower level.